x-pack/plugins/observability/public/pages/alerts/index.tsx

```diff
@@ -103,36 +102,62 @@ function AlertsPage() {
     ];
   }, [indexNames]);
 
-  const setWorkflowStatusFilter = useCallback(
-    (value: AlertWorkflowStatus) => {
-      setWorkflowStatus(value);
-    },
-    [setWorkflowStatus]
-  );
+  // Keep the Workflow status code commented (no delete) as requested: https://github.com/elastic/kibana/issues/117686
+
+  // const setWorkflowStatusFilter = useCallback(
+  //   (value: AlertWorkflowStatus) => {
+  //     setWorkflowStatus(value);
+  //   },
+  //   [setWorkflowStatus]
+  // );
 
   const onQueryChange = useCallback(
     ({ dateRange, query }) => {
       if (rangeFrom === dateRange.from && rangeTo === dateRange.to && kuery === (query ?? '')) {
         return refetch.current && refetch.current();
       }
-
       timefilterService.setTime(dateRange);
       setRangeFrom(dateRange.from);
       setRangeTo(dateRange.to);
       setKuery(query);
+      syncAlertStatusFilterStatus(query as string);
     },
     [rangeFrom, setRangeFrom, rangeTo, setRangeTo, kuery, setKuery, timefilterService]
   );
 
-  const addToQuery = useCallback(
-    (value: string) => {
-      let output = value;
-      if (kuery !== '') {
-        output = `${kuery} and ${value}`;
+  const syncAlertStatusFilterStatus = (query: string) => {
+    const [, alertStatus] = /\s*kibana\.alert\.status\s*:\s*"(.*?)"/.exec(query) || [];
+    if (!alertStatus) return;
+    switch (alertStatus.toLowerCase()) {
+      case 'active':
```

Comment:
Let's use the values from `@kbn/rule-data-utils/alerts_as_data_status`

Reply:
Done